platform/linux-generic/pktio/loop.c

```diff
@@ -172,6 +175,41 @@ static int loopback_recv(pktio_entry_t *pktio_entry, int index ODP_UNUSED,
 	return num_rx;
 }
 
+#define OL_TX_CHKSUM_PKT(_cfg, _capa, _proto, _ovr_set, _ovr) \
+	(_capa && _proto && (_ovr_set ? _ovr : _cfg))
+
+static inline int check_proto(void *l3_hdr,
+			      uint32_t l3_len,
+			      odp_bool_t *l3_proto_v4,
+			      uint8_t *l4_proto)
+{
+	uint8_t l3_proto_ver = _ODP_IPV4HDR_VER(*(uint8_t *)l3_hdr);
+
+	if (l3_proto_ver == _ODP_IPV4 && l3_len >= _ODP_IPV4HDR_LEN) {
+		_odp_ipv4hdr_t *ip = l3_hdr;
+
+		*l3_proto_v4 = 1;
+		if (!_ODP_IPV4HDR_IS_FRAGMENT(_odp_be_to_cpu_16(ip->frag_offset)))
+			*l4_proto = ip->proto;
```

Comment:
Checkpatch flags several line-too-long errors for this patch:
```
WARNING: line over 80 characters
#58: FILE: platform/linux-generic/pktio/loop.c:192:
+		if (!_ODP_IPV4HDR_IS_FRAGMENT(_odp_be_to_cpu_16(ip->frag_offset)))
WARNING: line over 80 characters
#113: FILE: platform/linux-generic/pktio/loop.c:279:
+						   pkt_hdr->p.flags.l3_chksum_set,
WARNING: line over 80 characters
#118: FILE: platform/linux-generic/pktio/loop.c:284:
+						   pkt_hdr->p.flags.l4_chksum_set,
WARNING: line over 80 characters
#123: FILE: platform/linux-generic/pktio/loop.c:289:
+						   pkt_hdr->p.flags.l4_chksum_set,
total: 0 errors, 4 warnings, 0 checks, 155 lines checked
NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.
0005-linux-gen-pktio-loop-support-IPv4-TCP-UDP-checksum-g.patch has style problems, please review.
```

Reply:
Fixed both files.